cluster-agent/utils/argocd_login_command_test.go

```diff
@@ -1,34 +1,31 @@
 package utils
 
 import (
-	"testing"
-
 	"github.com/argoproj/argo-cd/v2/pkg/apiclient/session"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 	"github.com/redhat-appstudio/managed-gitops/cluster-agent/utils/mocks"
-	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/mock"
 )
 
-func TestArgoCDLoginCommand(t *testing.T) {
-
-	// Ensure we are able to login to get a token
-
-	t.Parallel()
```

Comment:
I guess you discussed it in meetings and IIRC there was some approach suggested to execute ginkgo tests in parallel, is that right?

Reply:
Yes @jparsai I discussed this and we need to pass "ginkgo -p"  command to run ginkgo tests parallel

Reply:
Then it would be better if you mention it somewhere, maybe a comment at the starting of functions to inform others that these tests are expected to be executed parallelly.

Reply:
Yes @jparsai  added comment

Reply:
Is CI (PR Build) is also running it in parallel?

Reply:
I don't think so @jparsai